### PR TITLE
* docs/ref/uvh5_memo specify baseline direction

### PR DESCRIPTION
**Pull request by radonnachie**
## Description
Following a discussion on slack, this updates the UVH5 memo to reflect the baseline convention within the `uvw_array` definition, closely mirroring wording in the `uvdata_tutorial`. I've taken the liberty to extend the `conjugation convention` content to echo the definition... this may be contentious, happy to adjust.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation change (documentation changes only)
- [ ] Version change
- [ ] Build or continuous integration change

## Checklist:

- [ ] I have read the [contribution guide](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/main/.github/CONTRIBUTING.md).
- [ ] My code follows the code style of this project.

Bug fix checklist:
- [ ] My fix includes a new test that breaks as a result of the bug (if possible).
- [ ] All new and existing tests pass.
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/main/CHANGELOG.md).

New feature checklist:
- [ ] I have added or updated the docstrings associated with my feature using the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the tutorial to highlight my new feature (if appropriate).
- [ ] I have added tests to cover my new feature.
- [ ] All new and existing tests pass.
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/main/CHANGELOG.md).

Breaking change checklist:
- [ ] I have updated the docstrings associated with my change using the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] I have updated the tutorial to reflect my changes (if appropriate).
- [ ] My change includes backwards compatibility and deprecation warnings (if possible).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests pass.
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/main/CHANGELOG.md).

Documentation change checklist:
- [ ] Any updated docstrings use the [numpy docstring format](https://numpydoc.readthedocs.io/en/latest/format.html).
- [ ] If this is a significant change to the readme or other docs, I have checked that they are rendered properly on ReadTheDocs. (you may need help to get this branch to build on RTD, just ask!)

Version change checklist:
- [ ] I have updated the [CHANGELOG](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/main/CHANGELOG.md) to put all the unreleased changes under the new version (leaving the unreleased section empty).
- [ ] I have noted any dependency changes since the last version and will update the conda package build accordingly.

Build or continuous integration change checklist:
- [ ] If required or optional dependencies have changed (including version numbers), I have updated the readme to reflect this.
- [ ] If this is a new CI setup, I have added the associated badge to the readme and to references/make_index.py (if appropriate).

### CODE REVIEW

**Comment by radonnachie** [comment]
maybe while we're at it, the conjugation convention reference to `data_array` could be updated to `visdata`... maybe I'm missing something though...

File: docs/references/uvh5_memo.tex [hunk 1]
```diff
@@ -165,8 +165,10 @@ \subsection{Required Parameters}
 \item \textbf{Ntimes}: \textit{int} The number of time samples present in the
   data. (\textit{Ntimes})
 \item \textbf{uvw\_array}: \textit{float} An array of the uvw-coordinates
-  corresponding to each observation in the data. This is a two-dimensional array of
-  size (Nblts, 3). Units are in meters. (\textit{uvw\_array})
+  corresponding to each observation in the data. Baselines are specified as from
+  \verb+ant_1_array+ to \verb+ant_2_array+, inferring the position of antenna 2
```
Comment (plaplant) [inline docs/references/uvh5_memo.tex:169]:
```suggestion
  \verb+ant_1_array+ to \verb+ant_2_array+, implying the position of antenna 2
```

**Review by plaplant — COMMENTED** [review]
Thanks @radonnachie! I appreciate your help improving the documentation in the memo.

I just have one small wording change suggestion on this. In addition, we should update the date at the beginning of the memo to indicate that the memo was last updated ~today. Otherwise, this looks great!

File: docs/references/uvh5_memo.tex (hunk 1 above)
Comment (radonnachie) [reply docs/references/uvh5_memo.tex]: Happily... I was sure I'd gotten it right but in revision this feels like a bit of a grey area for nit-picking over it 🤷... I followed the following from an [IEEE guide (Sec. 3, Words Often Confused)](http://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE-Editorial-Style-Manual-for-Authors.pdf):
```
Imply: something suggested though not expressed.
Infer: something deduced from evidence.
```

Comment (plaplant) [reply docs/references/uvh5_memo.tex]: You're right, I'm definitely being pedantic, we could have left it the way it was. Thanks for making the change anyway.

**Review by plaplant — APPROVED** [review]
Looks great, thanks veyr much @radonnachie!

**Pull request merged**